### ISSUE TITLE
runtime.jst configuration.isStrictRequested() should not be called if it not exists

### ISSUE THREAD

**Issue by pke** [cucumber/cucumber-js#325]
The `configuration.isStrictRequested()` call should only be made if this function exists. Otherwise, the catch handler is called but the exception is still reported as first-chance in the Visual Studio IDE interrupts the program flow.

Simple fix:
````
isStrictRequested    = configuration.isStrictRequested && configuration.isStrictRequested();
```

**Comment by karthiktv006** [comment]
It call isStrictRequested in argument_parser.js:
```javascript
isStrictRequested: function isStrictRequested() {
      return self.getOptionOrDefault(ArgumentParser.STRICT_FLAG_NAME, ArgumentParser.DEFAULT_STRICT_FLAG_VALUE);
    }

ArgumentParser.DEFAULT_STRICT_FLAG_VALUE = false;
```

So, even if the value is not set it returns false

**Comment by pke** [comment]
@karthiktv006 what you mean?

**Comment by charlierudolph** [comment]
I personally want to remove the try/catch and just have it be
```
isStrictRequested = configuration.isStrictRequested();
```
Why would it ever not be defined?

**Comment by jbpros** [comment]
Fixed in 3651052
